Kudu.Console/Program.cs

```diff
@@ -162,12 +162,15 @@ private static int Main(string[] args)
                     deploymentManager.DeployAsync(gitRepository, changeSet: null, deployer: deployer, clean: false)
                         .Wait();
 
-                    string branch = settingsManager.GetBranch();
-                    ChangeSet changeSet = gitRepository.GetChangeSet(branch);
-                    IDeploymentStatusFile statusFile = deploymentStatusManager.Open(changeSet.Id);
-                    if (statusFile != null && statusFile.Status == DeployStatus.Success)
+                    if (autoSwapHander.IsAutoSwapEnabled())
```

Comment:
This part is just an optimization and not a functional change, right? Since `HandleAutoSwap` already checks if enabled.